packages/node/src/plugins/segmentio/index.ts

```diff
@@ -1,9 +1,10 @@
 import { CoreContext, CorePlugin } from '@segment/analytics-core'
 import { Publisher, PublisherProps } from './publisher'
+import { version } from '../../../package.json'
```

Comment:
Does this end up changing the structure of our dist build in a way that breaks out imports or is this already covered?

Reply:
correct, it was already covered by a previous import. If the structure changes, we'll also see an integration test failure.